### PR TITLE
preppin

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,3 @@
-<!-- AM: To Dos -->
-<!-- AM: What happens if when you call new on an object (not a constructor). Is it no prototype? -->
```
Comment (amaseda) [inline readme.md:2]: Oops, I shoulda gotten rid of these ages ago.

File: readme.md [hunk 2]
```diff
@@ -168,12 +139,12 @@ You'll notice that, at the moment, we have hard-coded models into the Grumbles c
...
 $ cd grumblr_rails_api
 $ bundle install
-$ rake db:create
-$ rake db:migrate
-$ rake db:seed
+$ rails db:create
+$ rails db:migrate
+$ rails db:seed
 $ rails s
 ```
```
Comment (amaseda) [inline readme.md:149]: Updated to Rails 5?

File: readme.md [hunk 3]
```diff
@@ -391,29 +305,14 @@ Let's update our index controller so that, instead of using hard-coded grumbles,
...
-
-```html
-<!-- index.html -->
-<script src="js/app.js"></script>
-<script src="js/grumbles/grumbles.js"></script>
-<script src="js/grumbles/index.controller.js"></script>
-<script src="js/grumbles/show.controller.js"></script>
-<script src="js/grumbles/grumble.factory.js"></script>
-```
+#### Create a Show Controller
```
Comment (amaseda) [inline readme.md:315]: Is there any code for this section?

File: readme.md (hunk 2 above)
Comment (jshawl) [reply readme.md:149]: 👍

File: readme.md (hunk 3 above)
Comment (jshawl) [reply readme.md:315]: nope!  gonna do another run through now n make sure im not mssin anythin

**Pull request merged**